apps/explorer/tsconfig.json

```diff
@@ -27,7 +30,11 @@
     "next-env.d.ts",
     ".next/types/**/*.ts",
     ".next-development/types/**/*.ts",
-    ".next-development-testnet-zen/types/**/*.ts"
+    ".next-development-testnet-zen/types/**/*.ts",
```

Comment:
not sure why this was added twice, you should be able to remove the duplicate and also the last ..dist one.